src/control/serve_http.rs

```diff
@@ -41,3 +55,9 @@ where
 
     log.future(fut)
 }
+
+impl ClientAddr {
+    pub fn into_socket_addr(&self) -> SocketAddr {
+        self.0
+    }
+}
```

Comment:
Thoughts on this vs `impl Into<SocketAddr> for ClientAddr` ?

Reply:
This began its life as `Into<SocketAddr>`, I recall there being type inference issues that required annotating it, so i thought I'd just add a function instead. It can be `Into` if you prefer.